data/XML/imbuements.xml

```diff
@@ -1,7 +1,7 @@
 <imbuements>
 	<base id="1" name="Basic" price="5000" protectionPrice="10000" percent="90" removecost="15000" duration="72000" />
 	<base id="2" name="Intricate" price="25000" protectionPrice="30000" percent="70" removecost="15000" duration="72000" />
-	<base id="3" name="Powerful" price="50000" protectionPrice="100000" percent="50" removecost="15000" duration="72000" />
+	<base id="3" name="Powerful" price="100000" protectionPrice="50000" percent="50" removecost="15000" duration="72000" />
```

Comment:
are you sure this is correct? it seems to me that it was inverted

Reply:
if you invert the values they will work but the case here is, to make it sounds as it is
the **price** is working as **protectionPrice** and vice versa

as previously said, if people dont care about naming it correctly, just inverting it will work. eg powerful, the price for it is 100k not 50k

protection price is 50k

what if you want the price to be 300k and 100k to make it 100% granted,  are you gonna set it where? 300k for price and 100k for protection price, right?
BUT, if you do it, 300k will be the price to make it 100% and 100k will be the price to make it..

hope you got it.
but as i said, if people understand that they need to just invert the price and having price as protectionPrice and vice versa, theres no need to update it.

its just to make sure it will work as it was made to.

Reply:
> if you invert the values they will work but the case here is, to make it sounds as it is the **price** is working as **protectionPrice** and vice versa
> 
> as previously said, if people dont care about naming it correctly, just inverting it will work. eg powerful, the price for it is 100k not 50k
> 
> protection price is 50k
> 
> what if you want the price to be 300k and 100k to make it 100% granted, are you gonna set it where? 300k for price and 100k for protection price, right? BUT, if you do it, 300k will be the price to make it 100% and 100k will be the price to make it..
> 
> hope you got it. but as i said, if people understand that they need to just invert the price and having price as protectionPrice and vice versa, theres no need to update it.
> 
> its just to make sure it will work as it was made to.
> 
> //edit(better than adding another reply Dunno if you did read the post and saw the images or just did read the code, but you'll see that they arent working as they should, the only one working is powerful just cuz its inverted (and it shouldnt be this way)

Ok, I got it. Just looked at the code.